### PR TITLE
Metada field security sharing docs

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Updated documentation for sharing samples fields with metadata restrictions.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] ~~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~~
* [ ] ~~Tests added (or description of how to test) for any new features.~~
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/_includes/tutorials/common/samples/copy-samples.md [hunk 1]
```diff
@@ -19,15 +19,21 @@ Once selected, you will be redirected to the new Share / Move Samples page.
 
 The first action is to select the destination project. Start typing the name of the destination project in the "Select a
 project to share samples with" input. As you type a list of projects will be presented, select the project you want.
+Once you select a project to share with, the "Next" button will become enabled, clicking on it will allow you to review
+the samples that were selcted to copy.
```
Comment (deepsidhu85) [inline doc/_includes/tutorials/common/samples/copy-samples.md:23]: Typo in `selected`

File: doc/_includes/tutorials/common/samples/copy-samples.md [hunk 2]
```diff
@@ -40,15 +46,40 @@ then select the checkbox "Remove samples from current project (move samples)"
...
 **NOTE: Both checkboxes cannot be selected at the same time.**
 
+### Metadata Restrictions
+
+![Share samples metadata restrictions.]({{ site.baseurl }}/images/tutorials/common/samples/share-metadata.png)
+
+Samples are shared/moved with all their corresponding metadata. This step allows you to review the fields that are
+included with the sample as well as review the metadata restriction level that will used on the destination project.
+
+If the metadata field does not exist in it will be set to the current level in the current project.
```
Comment (deepsidhu85) [inline doc/_includes/tutorials/common/samples/copy-samples.md:63]: Change `current level` to `current metadata restriction level`

File: doc/_includes/tutorials/common/samples/copy-samples.md [hunk 3]
```diff
@@ -40,15 +46,40 @@ then select the checkbox "Remove samples from current project (move samples)"
...
+
+![Share samples metadata restrictions.]({{ site.baseurl }}/images/tutorials/common/samples/share-metadata-current.png)
+
+In this example, metadata field `secondaryPfge` has a restrictin level of `Level 4`, the highest setting,
```
Comment (deepsidhu85) [inline doc/_includes/tutorials/common/samples/copy-samples.md:67]: Typo in `restriction` and change `highest setting` to `highest level`

File: doc/_includes/tutorials/common/samples/copy-samples.md [hunk 4]
```diff
@@ -40,15 +46,40 @@ then select the checkbox "Remove samples from current project (move samples)"
...
+and `birthDate` has a restriction level of `Level 1`, the lowest level. They both do not exist in the target project, so
+ey are initially set to that value. You can always set different levels in the destination project, **but please ensure
```
Comment (deepsidhu85) [inline doc/_includes/tutorials/common/samples/copy-samples.md:69]: Typo here `ey` should be `they`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looks good just a few comments

File: doc/_includes/tutorials/common/samples/copy-samples.md (hunk 1 above)
Comment (joshsadam) [reply doc/_includes/tutorials/common/samples/copy-samples.md]: Updated in 0e8f9a2a60ad5a105bb12339b85daf89fe3908f1

File: doc/_includes/tutorials/common/samples/copy-samples.md (hunk 2 above)
Comment (joshsadam) [reply doc/_includes/tutorials/common/samples/copy-samples.md]: Updated in 0e8f9a2a60ad5a105bb12339b85daf89fe3908f1

File: doc/_includes/tutorials/common/samples/copy-samples.md (hunk 3 above)
Comment (joshsadam) [reply doc/_includes/tutorials/common/samples/copy-samples.md]: updated in Updated in 0e8f9a2

Comment (joshsadam) [reply doc/_includes/tutorials/common/samples/copy-samples.md]: updated in Updated in 0e8f9a2

File: doc/_includes/tutorials/common/samples/copy-samples.md (hunk 4 above)
Comment (joshsadam) [reply doc/_includes/tutorials/common/samples/copy-samples.md]: Updated in 0e8f9a2

**Pull request merged**